src/lib/components/ui/ArticleList.svelte

```
@@ -1,129 +1,131 @@
+	import ArticleCard from './ArticleCard.svelte';
+	import { slide } from 'svelte/transition';
+	import { tick } from 'svelte';
+
+	const ARTICLES_PER_PAGE = 9;
+
+	const {
+		articles,
+		articlesPerPage = ARTICLES_PER_PAGE,
+		displayLoadMore = true,
+		title = 'Latest Research',
+		disableCategory = false
```

ESLint

[error] 'disableCategory' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)